config/v2/config.go

```diff
@@ -135,12 +135,15 @@ func (c *Config) Generate(r *rand.Rand, size int) reflect.Value {
 	}
 
 	randAWSProvider := func(r *rand.Rand, s int) *AWSProvider {
-		return &AWSProvider{
-			AccountID: randInt64Ptr(r, size),
-			Region:    randStringPtr(r, s),
-			Profile:   randStringPtr(r, s),
-			Version:   randStringPtr(r, s),
+		if r.Float32() < 0.5 {
```

Comment:
I wonder if this method should be renamed from `Generate` to `GenerateTestConfig` to mark explicitly

Reply:
Yeah probably.

Reply:
Actually we can't do that. test/quick relies on this method name

Reply:
https://github.com/chanzuckerberg/fogg/issues/249